.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,41 @@
+name: Bug report
+description: File a bug report to help us improve
+labels: ["bug", "needs-triage"]
```

```suggestion
labels: ["bug", "needs: triage"]
```

... and the label needs to be created, I guess.

---

Label still wasn't created. Maybe we can reuse `needs: assignee`?